io.github.Archeb.opentrace.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.github.Archeb.opentrace
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
this is EOL

Reply:
Upstream uses an old version of web dependency which doesn't exist in newer versions of sdk.

Reply:
you can build webkit2gtk/libsoup2, (porting to supported webkit2gtk is encouraged)

Reply:
https://github.com/search?q=org%3Aflathub+webkit2gtk-4.0&type=code

Reply:
So you are installing a binary that require dependencies that are not part of an ABI guarrantee.

You should build it from source instead, using a recent SDK.

Reply:
since the license is actually GPL-3.0